app/src/main/java/com/cypressworks/mensaplan/food/PlanAdapter.java

```diff
@@ -67,6 +74,9 @@ public View getView(final int position, View v, final ViewGroup parent) {
                 tag = (HeaderHolder) v.getTag();
             }
             final Line line = (Line) items.get(position);
+            if (firstLineName.equals(line.getName())) {
```

Comment:
I think we should do this by checking if `position == 0`

Reply:
I did that, but somehow 0 appeared multiple times which led to missing separators in the middle of the list. Commit 804bb86 has this functionality